maintainers/maintainer-list.nix

```diff
@@ -1417,6 +1417,12 @@
     githubId = 12958979;
     name = "Mika Naylor";
   };
+  avakhrenev = {
```

Comment:
there is probably no need here, but usually adding yourself as a maintainer is done in a separate commit, just in case a commit need to be reverted.

Reply:
Should I remove myself and make a separate PR?

Reply:
it's just a detail, I'm mentioning it here to make sure the information is not lost, but honestly, It's a detail.